slang-gfx.h

```diff
@@ -1424,6 +1424,8 @@ class IDevice: public ISlangUnknown
         DeviceType deviceType = DeviceType::Default;
         // Name to identify the adapter to use
         const char* adapter = nullptr;
+        // The device handle. If 0, a device will need to be created.
```

Comment:
This is OK for now, but for vulkan we may need to pass in both a `VkInstance` and `VkDevice`, so perhaps we can use something like:
```
uint64_t existingDeviceHandles[2];
```
Or even better we can define a `struct` type:
```
struct ExistingDeviceHandle
{
      uint64_t values[2];
};
```

Reply:
Done. I'll leave the helpers you mentioned for later.